src/init.js

```diff
@@ -61,8 +69,8 @@ const initUser = () => {
   };
 };
 
-export default ({ clientSideId, dispatch, flags, user, subscribe, options }) => {
-  initFlags(flags, dispatch);
+export default ({ clientSideId, dispatch, flags, camelCased, user, subscribe, options }) => {
```

Comment:
What do you think of calling `camelCased` => `useCamelCaseFlagKeys` instead?

Also, the default value of this can be set once here so we can avoid having to default it again elsewhere:

```js
// set default value for camel case here at the root
export default ({ clientSideId, dispatch, flags, useCamelCaseFlagKeys = true, user, subscribe, options }) => {
  initFlags(flags, dispatch, useCamelCaseFlagKeys);
  // rest of code
```

Reply:
Can you also please update the readme to reflect this new option? Thank you.

Reply:
`useCamelCaseFlagKeys` is fine with me 👍. I'll also update the docs 🙂